src/overlays/actors/ovl_En_Al/z_en_al.c

```diff
@@ -159,30 +163,30 @@ Actor* func_80BDE1A0(EnAl* this, PlayState* play, u8 arg0, s16 arg1) {
     return foundActor;
 }
 
-void func_80BDE250(EnAl* this) {
-    this->skelAnime.playSpeed = this->unk_4C8;
+void EnAl_UpdateAnim(EnAl* this) {
+    this->skelAnime.playSpeed = this->animPlaySpeed;
     SkelAnime_Update(&this->skelAnime);
 }
```

Comment:
This is another case where I'm not sure if we should go with `_UpdateAnim` (to parallel `_ChangeAnim`) or to keep it as `_UpdateSkelAnime`. Whenever an actor stores `this->skelAnime.playSpeed` in `_UpdateAnim` via `this->animPlaySpeed;`, it usually is restored in `_UpdateAnim`, so often these two functions are tied

Reply:
Personally I like keeping `_UpdateSkelAnime` since it does end up calling `SkelAnime_Update`.